packages/core/src/server/socketServer.ts

```diff
@@ -17,6 +21,7 @@ export class SocketServer {
   private readonly options: DevConfig;
 
   private stats?: Stats;
+  private entryPointsChunks?: string[];
```

Comment:
Can we rename it to `initialChunks`

```suggestion
  private initialChunks?: string[];
```